### PR TITLE
Release v1.4.0

### PR DESCRIPTION
**Pull request by carlosalberto**
cc @tigrannajaryan

### CODE REVIEW

**Comment by Oberon00** [comment]
I think @reyang mentioned that he'd like to have #1733 in this release. Is this feasible?

**Review by tigrannajaryan — APPROVED** [review]
LGTM.
We will need to merge https://github.com/open-telemetry/opentelemetry.io/pull/599 after the release.

**Comment by carlosalberto** [comment]
Ping @reyang

**Comment by reyang** [comment]
> Ping @reyang

#1733 is ready to be merged (it is an editorial change).

**Review by reyang — APPROVED** [review]
LGTM.

**Comment by carlosalberto** [comment]
Let's merge/release on early Monday, to avoid weekend surprises.

**Review by bogdandrutu — APPROVED** [review]

**Review by arminru — APPROVED** [review]

**Comment by jsuereth** [comment]
Would you be ok merging https://github.com/open-telemetry/opentelemetry-specification/pull/1728 before releasing 1.4?

At a minimum, we want OTLP metrics marked stable in 1.4

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,28 @@ release.
...
+
+### Semantic Conventions
+
+### Compatibility
+
+### OpenTelemetry Protocol
+
+### SDK Configuration
+
+## v1.4.0 (2021-06-04)
```
[the remark this answers is not in the document]
Comment (carlosalberto) [reply CHANGELOG.md]: Uff, yes. Indeed, we had two schema-related entries in the 1.3 section :( Fixed.

**Pull request merged**